src/query/service/src/interpreters/interpreter_copy_v2.rs

```diff
@@ -135,17 +187,33 @@ impl CopyInterpreterV2 {
     ) -> Result<()> {
         tracing::info!("upsert_copied_files_info: {:?}", copy_stage_files);
 
-        if !copy_stage_files.is_empty() {
-            let req = UpsertTableCopiedFileReq {
-                table_id,
-                file_info: copy_stage_files.clone(),
-                expire_at: None,
-            };
-            let catalog = self.ctx.get_catalog(catalog_name)?;
-            catalog
-                .upsert_table_copied_file_info(&self.ctx.get_tenant(), database_name, req)
+        if copy_stage_files.is_empty() {
+            return Ok(());
+        }
+
+        let mut do_copy_stage_files = BTreeMap::new();
+        for (file_name, file_info) in copy_stage_files {
+            do_copy_stage_files.insert(file_name.clone(), file_info);
+            if do_copy_stage_files.len() > MAX_QUERY_COPIED_FILES_NUM {
```

Comment:
`copy_stage_files ` is type of `BTreeMap`, has no `chunks` method.

Reply:
It's better to collect them to vec?

Reply:
this function collect data to RPC `UpsertTableCopiedFileReq`, and `UpsertTableCopiedFileReq.file_info` is a `BTreeMap`.

So, if collect as a vector, it will convert to `UpsertTableCopiedFileReq.file_info` as a `BTreeMap`, which first `BtreeMap` to `Vector` and then `Vector` to `BtreeMap`, will cause more time and conversion.